build.gradle

```diff
@@ -140,6 +165,17 @@ subprojects {
     }
 }
 
+project(":mocca-http-client-tests") {
+    sonarqube {
+        skipProject = true
+    }
+}
+project(":mocca-micrometer") {
```

Comment:
Let's not skip this please. If we skipped it because it doesn't have unit tests, can you add a very simple unit test so we don't have to skip it? It can be a placeholder unit test.

Reply:
Let's skip the functional test module too please.